Misc/NEWS.d/next/Library/2024-07-17-17-51-25.gh-issue-121331.8Gk6LP.rst

```diff
@@ -0,0 +1 @@
+inspect.getsource now works for genexp code objects, no longer only returning the first line of a multi-line expression, and now only returning the code of the generator expression itself, not the whole line of code it is created on, allowing for disambiguation when multiple generator expressions are defined on the same line.
```

Comment:
```suggestion
:meth:`inspect.getsource` now works for genexp code objects, no longer only returning the first line of a multi-line expression, and now only returning the code of the generator expression itself, not the whole line of code it is created on, allowing for disambiguation when multiple generator expressions are defined on the same line.
```

Reply:
Indeed, you're right. The `func` directive is more appropriate here.

Reply:
OK, I'll change back to `func`